modules/install-sno-generating-the-install-iso-manually.adoc

```diff
@@ -86,39 +86,42 @@ $ curl -L $ISO_URL -o rhcos-live.iso
 apiVersion: v1
 baseDomain: <domain> <1>
 compute:
+- architecture: amd64 <2>
 - name: worker
-  replicas: 0 <2>
+  replicas: 0 <3>
 controlPlane:
+  architecture: amd64
   name: master
-  replicas: 1 <3>
+  replicas: 1 <4>
 metadata:
-  name: <name> <4>
-networking: <5>
+  name: <name> <5>
+networking: <6>
   clusterNetwork:
   - cidr: 10.128.0.0/14
     hostPrefix: 23
   machineNetwork:
-  - cidr: 10.0.0.0/16 <6>
+  - cidr: 10.0.0.0/16 <7>
   networkType: OVNKubernetes
   serviceNetwork:
   - 172.30.0.0/16
 platform:
   none: {}
 bootstrapInPlace:
-  installationDisk: /dev/disk/by-id/<disk_id> <7>
-pullSecret: '<pull_secret>' <8>
+  installationDisk: /dev/disk/by-id/<disk_id> <8>
+pullSecret: '<pull_secret>' <9>
 sshKey: |
-  <ssh_key> <9>
+  <ssh_key> <10>
 ----
 <1> Add the cluster domain name.
-<2> Set the `compute` replicas to `0`. This makes the control plane node schedulable.
-<3> Set the `controlPlane` replicas to `1`. In conjunction with the previous `compute` setting, this setting ensures the cluster runs on a single node.
-<4> Set the `metadata` name to the cluster name.
-<5> Set the `networking` details. OVN-Kubernetes is the only allowed network plugin type for single-node clusters.
-<6> Set the `cidr` value to match the subnet of the {sno} cluster.
-<7> Set the path to the installation disk drive, for example, `/dev/disk/by-id/wwn-0x64cd98f04fde100024684cf3034da5c2`.
-<8> Copy the {cluster-manager-url-pull} and add the contents to this configuration setting.
-<9> Add the public SSH key from the administration host so that you can log in to the cluster after installation.
+<2> Set the architecture to `arm64` for 64-bit ARM or `amd64` for 64-Bit x86 architectures. This needs to be set explicitly to the target host architecture otherwise it picks up the binary's compiled architecture.
```

Comment:
In this doc[1] we have a table that provides description for architecture. It says:
"Defines the host architecture for your cluster. Valid values are amd64 or arm64."

I'd suggest either going with the same or adjusting to:
Set the architecture to `arm64` for 64-bit ARM or `amd64` for 64-Bit x86 architectures. This needs to be set explicitly to the target host architecture.


[1]
https://docs.openshift.com/container-platform/4.14/installing/installing_bare_metal_ipi/ipi-install-installation-workflow.html#additional-install-config-parameters_ipi-install-installation-workflow

Reply:
Thank you @achuzhoy with this update are you OK to give it a LGTM and and can check the QE box?

Reply:
added lgtm - can't find the qe box.:)